spec/features/results/response_form_tree_handling_spec.rb

```diff
@@ -2,35 +2,41 @@
 
 require "rails_helper"
 
-feature "response form tree handling", js: true do
+feature "response form tree handling", js: true, database_cleaner: :all do
```

Comment:
Why was this needed?

Reply:
As for file names, perhaps like response_form_question_types_and_tree_handling, response_form_file_upload, response_form_location_picker, etc. Also let's move the response form specs that are in the root of `spec/features` into the same directory and try to make them consistent?

Reply:
Looks like `database_cleaner: :all` is not needed